src/frontend/src/flows/addDevice/manage/pollForTentativeDevice.ts

```diff
@@ -32,17 +32,14 @@ const pollForTentativeDeviceTemplate = ({
       <li>
         Open
         <em class="c-tooltip">
-          <strong class="t-strong">${LEGACY_II_URL}</strong>
+          <strong data-role="add-device-link" class="t-strong t-wrap"
+            >${link}</strong
```

Comment:
would it be handy ux wise to provide the link has a link?

The device can be a device on the same laptop

Reply:
Right; I'll replace this with a QR code very soon anyway, but the idea is that you anyway need to open the link on _another_ device so there's no real point of having an `<a>` here. Does that make sense? Or do you see a use case for opening the link in the same browser?

Reply:
I was more thinking on copy/pasting the link easily but without link, it's also more explicit. works for me as it, anyhow, not my call 😉

Reply:
curiosity question it was

Reply:
A very good one too; I did wonder about this too